.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build
+
+on:
+  push
+env:
+  NODE_VERSION: '14.15.3'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ env.NODE_VERSION }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+    - run: |
+        npm install -g ajv-cli@3.0.0
+        chmod +x ./test.sh
```

Comment:
Spricht etwas dagegen das execute-Flag direkt zu commiten? Dann könnte man es sich sparen das jedesmal zu setzen

Reply:
Da ich Windows nutze, hat es sich bisher als die 'sichere' Variante herausgestellt, die nötigen file permissions in die Github Actions zu packen. Also wenn ein Windows User die Datei bearbeitet, ist es möglich, dass das execute flag wieder verschwindet.